samples/profiler/README.rst

```diff
@@ -41,7 +41,8 @@ Testing
 =======
 
 After programming the sample to your board, you can test it by running the script ``real_time_plot.py`` (located under :file:`scripts/profiler`).
-As arguments, pass the csv and json file names that should be used to store the data (for example, run ``real_time_plot.py data.csv data.json``).
+As an argument, pass the test preset name (for example, run ``real_time_plot.py test_name``).
```

Comment:
How do I know what name I should pass?

Reply:
You can pass any name here. It will be used later for naming the files with stored data.
For example if you call pass: `test123`, the files will be named `test123.csv` and `test123.json`

Reply:
I can change it to `pass the arbitrary test preset name`. Will this be ok ?